frontend/src/components/main/Main.tsx

```diff
@@ -4,17 +4,54 @@
 
 import React from "react";
 
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+
 import authService from "../../services/auth";
+import Events from "./events/Events";
+
+const useStyles = makeStyles(() =>
```

Comment:
Please consider moving styles to separate file e.g. `styles.tsx` kept next to component main file

Reply:
Ok, but I'll do it in separate branch. Will be more clear that way.